rides-service/src/test/java/com/java/akdev/ridesservice/util/TestSetUps.java

```diff
@@ -0,0 +1,69 @@
+package com.java.akdev.ridesservice.util;
+
+import com.java.akdev.ridesservice.dto.RideCreateDto;
+import com.java.akdev.ridesservice.dto.RideReadDto;
+import com.java.akdev.ridesservice.dto.RideUpdateDto;
+import com.java.akdev.ridesservice.entity.Ride;
+import lombok.experimental.UtilityClass;
+
+import java.util.UUID;
+
+@UtilityClass
+public class TestSetUps {
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class TestSetUps {
```
Не рекомендуется использовать экспериментальные функции. Посмотри, какие ограничения накладывает использование `@UtilityClass`.
Обычно достаточно сделать утильный класс с приватным дефолтным конструктором, а также сделать класс final, чтобы его нельзя было расширить.